### DISCUSSION TITLE
ratio calculation

### DISCUSSION THREAD

**Discussion by Teloriun** [openscopeproject/TrguiNG#83] [General]
hi

me again :(

Is it intentional that the ratio is calculated comparing full file size and uploaded data rather than data downloaded and uploaded?

eg.  multi-file torrent where only part is download

I never noticed before as it's my first partial file download.

![image](https://github.com/openscopeproject/TrguiNG/assets/13600266/428cfebb-9785-4282-b276-066d4126386f)

**Comment by qu1ck** [comment]
It is intentional but not on my part, the app just shows what the server is reporting.

See https://github.com/transmission/transmission/issues/4991 and issues linked within for context.

**Reply by Teloriun** [reply]
Appreciate the link to the information.

Seems a bit silly in my opinion...  any scope for an additional feature for a 'real' ratio (optional to users of course, default would be as is) ?

**Reply by qu1ck** [reply]
I could add a separate `U/D` column that will calculate ratio directly but keep in mind that it will not affect any of server side logic that stops torrents based on ratio for example.

**Reply by Teloriun** [reply]
If you have capacity to add it, that would be great. If not, the explanation of the change to transmission 4.x will suffice.  thanks.

**Reply by qu1ck** [reply]
New column is available in [v0.10.0](https://github.com/openscopeproject/TrguiNG/releases/tag/v0.10.0).
